util/batch-instance-whitelist.py

```diff
@@ -113,8 +116,18 @@ def main(args):
     parser.add_argument("--dryrun", type=str, help="Doesn't push anything to S3, just outputs", required=True)
     args = parser.parse_args()
 
+    if args.partition == "commercial":
```

Comment:
I would move this selection of the region to the get_all_aws_regions function

Reply:
done